doc/user_guide/ide_integration/visual-studio-code.rst

```diff
@@ -0,0 +1,62 @@
+.. _visual-studio-code:
+
+Integrate Pylint with Visual Studio Code
+========================================
+
+See external documentation:
+
+- Linting python: https://code.visualstudio.com/docs/python/linting
+- Pylint Extension: https://marketplace.visualstudio.com/items?itemName=ms-python.pylint
```

Comment:
Added the external doc here, I'm hesitating to simply remove our own doc and only link one of the link.

Reply:
I'd be okay with removing it. There is a bit of duplication with other sections as well.

Reply:
I removed the two that had better doc elsewhere.